.github/workflows/create_jira_workflow.yml

```diff
@@ -26,6 +26,11 @@ jobs:
             ISSUE_TYPE="Story"
           fi
 
+           # Find active sprint ID
+          SPRINT_ID=$(curl -s -u "yevhenii.oros@droidsonroids.pl:${JIRA_API_TOKEN}" \
```

Comment:
mail @yevoros ma być w publiczmym repo?
A jak odejdzie z DoR to co? jego token przestanie hulać.
Trzeba to ogarnąć na automatyzacjach jiry albo jeśli się naprawdę nie da, to na userze bota.